conf/pvers.c

```diff
@@ -63,8 +63,10 @@ struct arr_int_pos {
 	uint32_t       ap_pos;
 };
 
+#ifdef ENABLE_ALT_PVERS
```

Comment:
It is preferred to have a conf (conf.xc) option, instead of this build time option.
@cc @nikitadanilov

Reply:
Ok, I think, if Hare don't generate formulaic pvers, we can probably just return the actual pver from conf_pver_find_locked() instead of -ENOENT?

@mssawant, what do you think?